x-pack/plugins/lens/public/datatable_visualization/components/columns.tsx

```diff
@@ -211,10 +213,24 @@ export const createGridColumns = (
     const currentAlignment = alignments && alignments[field];
     const alignmentClassName = `lnsTableCell--${currentAlignment}`;
 
+    const columnStyle =
+      headerRowHeight === 'auto' || headerRowHeight === 'custom'
+        ? {
+            whiteSpace: 'normal' as const,
+            display: '-webkit-box',
```

Comment:
If it's totally necessary this is fine, but if we can let's use emotion for this so we don't have to do prefixing ourselves.